jest.setup.js

```
@@ -2,6 +2,8 @@ const fs = require('fs');
 
 // Configure JSDOM environment for client tests
 if (typeof window !== 'undefined') {
+  require('@testing-library/jest-dom');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)